GetandTake/Pages/Product.cshtml.cs

```diff
@@ -1,22 +1,30 @@
+using GetandTake.Configuration;
 using GetandTake.Models.DTOs.ListDTO;
-using GetandTake.Services.Abstracts;
+using GetandTake.Services.Abstract;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace GetandTake.Pages;
 
 public class ProductModel : PageModel
 {
+
     private readonly IProductService _productService;
 
+    private readonly IConfiguration Configuration;
+
     public IEnumerable<ProductsDTO> Products { get; private set; }
 
-    public ProductModel(IProductService productService)
+    public int AmountOfProduct { get; private set; }
+
+    public ProductModel(IProductService productService, IConfiguration configuration)
     {
         _productService = productService;
+        Configuration = configuration;
     }
 
     public void OnGet()
     {
-        Products = _productService.GetAll();
+        AmountOfProduct = Convert.ToInt32(Configuration.GetSection("MaximumAmountOfProduct").Value);
```

Comment:
Injection of `IConfiguration` has sense, but, when your app is growing and you should store quite a lot of appsettings it is really easier for you to create some separate object that will store all these env values. For example, let's take a look at my solution [here](https://github.com/dotnet-fizzyy/Diploma/blob/master/WebAPI/WebAPI/Core/Configuration/AppSettings.cs#L3). You are declaring class with all config sections in you `appsettings.json`. On startup you should read these sections like in [this](https://github.com/dotnet-fizzyy/Diploma/blob/master/WebAPI/WebAPI/Startup/Startup.cs#L102) example and you properties will be automatically binded to values from app settings. After that, you should only register your object with appsettings with singleton scope (since it is used across your app) like [here](https://github.com/dotnet-fizzyy/Diploma/blob/9e7471ef8e545f26de578408bdc45c238a94de06/WebAPI/WebAPI/Startup/Configuration/ServiceExtensions.cs#L22) and feel free to use it across app :)